### DISCUSSION TITLE
we weren't able to approve your application.

### DISCUSSION THREAD

**Discussion by sbendarsky** [community/community#56839] [GitHub Education]
### Hi everyone,

I reached out for assistance earlier regarding a problem I encountered. I visited my college and obtained a translated student ID, which was previously in Hebrew. The new version is now in English. However, when I tried to verify it, I received an error message stating: "Unfortunately, your educational discount request for [@sbendarsky](https://github.com/sbendarsky) couldn't be approved." I also have an official transcript from my college, but the system isn't accepting it. I really need help with this situation.
BTW I've already submitted 27 requests, it might be the problem ;3

**Comment by sbendarsky** [comment]
well I think my account is on cooldown or some sort shadow ban. 
I tried everything... saw some suggestions of logging out and using incognito even using a different network...
help please @samus-aran

**Comment by Ntshangase** [comment]
Hey @sbendarsky 

It will be easier to assist you if you provided a screenshot of your rejection reason
have you tried :
 - submitting both the original version with the translated version next to it?
 - making sure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
 - If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) it's recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
- btw submitting the 27 request is not a problem, you can still get approved.

**Reply by sbendarsky** [reply]
the answer I got is:
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 1, 2023 for [@sbendarsky](https://github.com/sbendarsky).

Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

**Comment by sbendarsky** [comment]
I finally got approved, hopefully someone that faces the same problem will just have the patience that I had.

**Reply by miccio-dk** [reply]
Hi Stav,
thanks for the hopeful comment :) I am currently facing a similar issue.
I was wondering, did you get your application approved by continuing to submit alternative proofs or did they look at the rejected ones and eventually approved it?

**Reply by sbendarsky** [reply]
> Hi Stav,
> 
> thanks for the hopeful comment :) I am currently facing a similar issue.
> 
> I was wondering, did you get your application approved by continuing to submit alternative proofs or did they look at the rejected ones and eventually approved it?

My college translated my student id, and then I submitted the id and I got approved.
